test/cpp/qps/scenario_generator_helper.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+
+# Copyright 2021 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import json
+import os
+import sys
+
+import yaml
+
+run_tests_root = os.path.abspath(
+    os.path.join(os.path.dirname(sys.argv[0]), '../../../tools/run_tests'))
+sys.path.append(run_tests_root)
+
+import performance.scenario_config as scenario_config
+
+_COPYRIGHT = """# Copyright 2021 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+
+
+def _mutate_scenario(scenario_json):
+    # tweak parameters to get fast test times
+    scenario_json = dict(scenario_json)
+    scenario_json['warmup_seconds'] = 0
+    scenario_json['benchmark_seconds'] = 1
+    outstanding_rpcs_divisor = 1
+    if scenario_json['client_config'][
+            'client_type'] == 'SYNC_CLIENT' or scenario_json['server_config'][
+                'server_type'] == 'SYNC_SERVER':
+        outstanding_rpcs_divisor = 10
+    scenario_json['client_config']['outstanding_rpcs_per_channel'] = max(
+        1,
+        int(scenario_json['client_config']['outstanding_rpcs_per_channel'] /
```

Comment:
I think this field requires an int? If so, you'll want the integer division operator: `a // b`. This is one of the big differences in behavior between Python 2 and Python 3.